CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ DEPENDENCIES:
 
 * deps: bumps github.com/urfave/cli/v2 from 2.23.7 to 2.24.1 ([#1180](https://github.com/cloudflare/cloudflare-go/issues/1180))
 
+ENHANCEMENTS:
```

Comment:
this is a generated file and will be overwritten on next merge. to add a changelog entry, check out the docs at https://github.com/cloudflare/cloudflare-go/blob/master/docs/changelog-process.md.